festim/hydrogen_transport_problem.py

```diff
@@ -1,16 +1,7 @@
-import dolfinx
-from dolfinx import fem
-from dolfinx.nls.petsc import NewtonSolver
-import basix
-import basix.ufl
-import ufl
-from mpi4py import MPI
-import numpy as np
-import tqdm.auto as tqdm
-import festim as F
-
-
-class HydrogenTransportProblem:
+from .problem import *
```

Comment:
You should avoid this as much as possible.

```suggestion
from festim import Problem
```

Reply:
getting this error when I try to do this

```
ImportError: cannot import name 'ProblemBase' from partially initialized module 'festim' (most likely due to a circular import) (/home/rekomodo/festim/FESTIM/festim/__init__.py)
```

I'm guessing that we can't avoid it here since we're trying to import the module to define the module